src/optim/FishLeg/fishleg.py

```diff
@@ -258,11 +233,12 @@ def update_aux(self) -> None:
         """
         self.aux_opt.zero_grad()
 
-        data_x, data_y = next(iter(self.aux_dataloader))
-        data_x, data_y = data_x.to(self.device), data_y.to(self.device)
+        data_x, _ = next(iter(self.aux_dataloader))
```

Comment:
excellent, just  remember to change the dataloader call in the other scripts to include the collate_fn() that we talked about